xbmc/filesystem/MythSession.cpp

```diff
@@ -358,14 +358,14 @@ void CMythSession::SetSeasonAndEpisode(const cmyth_proginfo_t &program, int *sea
   return;
 }
 
-CMythSession::CMythSession(const CURL& url) : CThread("MythSession")
+CMythSession::CMythSession(const CURL& url) : CThread("MythSession"):
+  m_hostname(url.GetHostName()),
+  m_username(url.GetUserName() == "" ? MYTH_DEFAULT_USERNAME : url.GetUserName()),
+  m_password(url.GetPassWord() == "" ? MYTH_DEFAULT_PASSWORD : url.GetPassWord())
```

Comment:
Forgot a "check", sorry. Inits are done in the order the members are declared (in the header). If the order here is different the compiler prints a warning. I've checked it myself and it's ok ;)